packages/react-api/src/Api.tsx

```diff
@@ -18,10 +18,9 @@ import { formatBalance, isTestChain } from '@polkadot/util';
 import { setSS58Format } from '@polkadot/util-crypto';
 import addressDefaults from '@polkadot/util-crypto/address/defaults';
 
-import typesChain from './overrides/chain';
```

Comment:
No need to remove these two lines, so that we could ensure minimum unnecessary changes/conflicts from polkadot/apps

Reply:
have put it back

Reply:
May need to use it in `api = new ApiPromise({ provider, registry, signer, typesChain, typesSpec });` at line 129?

Reply:
Otherwise, each time when we rebase from polkadot, we have to resolve the conflict here, thats my concern.

Reply:
no.. we need to use types specific to our chain.. so I use the types as 
api = new ApiPromise({ provider, registry, signer, types: { ...CENNZRuntimeTypes as any } });

https://polkadot.js.org/api/start/types.extend.html#node-and-chain-specific-types
Single API object can be used to connect to different types of nodes or chains.

Reply:
Gotcha.